airflow/models/dagrun.py

```diff
@@ -419,6 +420,17 @@ def verify_integrity(self, session=None):
                     1, 1)
                 ti = TaskInstance(task, self.execution_date)
                 session.add(ti)
+                session.commit()
+
+                # ToDo: Store only Last X number (maybe 10 or 100) TIs for a task
```

Comment:
```suggestion
                # TODO: Store only Last X number (maybe 10 or 100) TIs for a task
```

(Just so they are all the same in the codebase and a `grep TODO` will find it.